src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -133,7 +133,8 @@ class_exists(__NAMESPACE__.'\TestingCaseMismatch', true);
      */
     public function testFileCaseMismatch()
     {
-        if (!file_exists(__DIR__.'/Fixtures/CaseMismatch.php')) {
+        //Cannot be tested if the file loads (there is is casemismatch.php in the filesystem)
+        if (file_exists(__DIR__.'/Fixtures/CaseMismatch.php')) {
```

Comment:
This doesn't look right. The test will no be executed on case-sensitive filesystems as the file could not be found and will then fail later (see the Travis builds).